### PR TITLE
[importer] Extend README.md

### PR DESCRIPTION
**Pull request by trasc**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Add more information about the importer flags.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: cmd/importer/README.md [hunk 1]
```diff
@@ -62,6 +62,41 @@ It's done providing a yaml mapping file name as `--queuemapping-file` argument,
...
+```
+
+- At least one `namespace` needs to be specified
+- `queuelabel` and `queuemapping` should always be used together.
+- One and only one of `queuemapping` and `queuemapping-file` should always be provided.
+
+
 ### Import
 
 After which, if `--dry-run=false` was specified, for each selected Pod the importer will:
```
Comment (alculquicondor) [inline cmd/importer/README.md:102]:
```suggestion
When running the importer, if `--dry-run=false` was specified, for each selected Pod the importer will:
```

File: cmd/importer/README.md [hunk 2]
```diff
@@ -62,6 +62,41 @@ It's done providing a yaml mapping file name as `--queuemapping-file` argument,
 - The rules are evaluated in order.
 - `skip: true` can be used to ignore the pods matching a rule.
 
+#### Other flags
+
+After building the executable the full list of supported flags can be retrieve by:
```
Comment (alculquicondor) [inline cmd/importer/README.md:67]:
```suggestion
After building the executable, the full list of supported flags can be retrieved by:
```

Comment (trasc) [reply cmd/importer/README.md]: done

File: cmd/importer/README.md (hunk 1 above)
Comment (trasc) [reply cmd/importer/README.md]: done

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**